recipe/build-basemap.sh

```diff
@@ -7,11 +7,18 @@ echo ""
 # Ensure our GEOS will be used.
 export GEOS_DIR="${PREFIX}"
 
-# Temporary workaround when cross-compiling.
-# if [ "${CONDA_BUILD_CROSS_COMPILATION}" = "1" -a "${PKG_NAME}" = "basemap" ]; then
-#     rm ${BUILD_PREFIX}/bin/python
-#     ln -sf ${PREFIX}/bin/python ${BUILD_PREFIX}/bin/python
-# fi
+# Temporary workaround when cross-compiling the data packages.
+if [ "${CONDA_BUILD_CROSS_COMPILATION}" = "1" -a "${PKG_NAME}" != "basemap" ]; then
+    echo "Debugging info":
+    echo "- PYTHON: ${PYTHON}"
+    echo "- Python: $(command -v python)"
+    echo "- BUILD_PREFIX: ${BUILD_PREFIX}"
+    echo "$(ls -la ${BUILD_PREFIX} || true)"
+    echo "- PREFIX: ${PREFIX}"
+    echo "$(ls -la ${PREFIX} || true)"
+    rm -f ${BUILD_PREFIX}/bin/python
+    ln -sf ${PREFIX}/bin/python ${BUILD_PREFIX}/bin/python
```

Comment:
You won't find python in the build prefix if you do not have the build section of the recipe install python.